src/discourseGraphsMode.ts

```diff
@@ -494,6 +494,17 @@ const initializeDiscourseGraphsMode = async (args: OnloadArgs) => {
                   Panel: TextPanel,
                 },
                 // @ts-ignore
+                {
+                  title: "open in sidebar",
+                  description:
+                    "Whether or not to open nodes in the sidebar when created",
+                  Panel: FlagPanel,
+                  options: {
+                    onChange: onPageRefObserverChange(previewPageRefHandler),
+                  },
+                  defaultValue: true,
```

Comment:
What's a good way to handle backwards compatibility for this?  @dvargas92495

Reply:
something like this?

add `extensionAPI.settings.set("sidebar-default-added", true);` to the switch

https://github.com/RoamJS/query-builder/blob/2b54dea77c1b6096108b73931dd1b4ddb83394be/src/index.ts#L357-L376

and to this to `runExtension`

```javascript
  if (
      extensionAPI.settings.get("discourse-graphs") &&
      !extensionAPI.settings.get("sidebar-default-added")
    ) {
      addSidebarFlagDefault();
      extensionAPI.settings.set("sidebar-default-added", true);
    }
```    

hmm, feels clunky ...

I guess I could reverse the flag to "check to **not** open in sidebar"